### PR TITLE
[Contracts] [PHPUnitBridge] Document the "symfony/deprecation-contracts"

### PR DESCRIPTION
**Pull request by noniagriconomie**
When reading this doc, I remembered this package, i think it can be useful to document it here

### CODE REVIEW

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -165,6 +165,9 @@ Deprecation notices can be triggered by using::
 
     @trigger_error('Your deprecation message', E_USER_DEPRECATED);
 
+You can also require the `symfony/deprecation-contracts` that provide the global
```
Comment (OskarStark) [inline components/phpunit_bridge.rst]:
```suggestion
You can also require the ``symfony/deprecation-contracts`` that provide the global
```

File: components/phpunit_bridge.rst [hunk 2]
```diff
@@ -165,6 +165,9 @@ Deprecation notices can be triggered by using::
...
+function `trigger_deprecation()` for this usage.
```
Comment (OskarStark) [inline components/phpunit_bridge.rst]:
```suggestion
``trigger_deprecation()`` function for this usage.
```

I would rather use:
```rst
:method:`\Symfony\........::trigger_deprecation`
```

here

Comment (noniagriconomie) [reply components/phpunit_bridge.rst]: let me find an example in the codebase @OskarStark :)

Comment (noniagriconomie) [reply components/phpunit_bridge.rst]: hum in fact there is no namespace etc, it is a global func, so what path should i write?

Comment (OskarStark) [reply components/phpunit_bridge.rst]: Lets keep it like this then, thanks!

File: components/phpunit_bridge.rst [hunk 3]
```diff
@@ -165,6 +165,9 @@ Deprecation notices can be triggered by using::
 
     @trigger_error('Your deprecation message', E_USER_DEPRECATED);
 
+You can also require the ``symfony/deprecation-contracts`` that provides the global
+``trigger_deprecation()`` function for this usage.
```
Comment (OskarStark) [inline components/phpunit_bridge.rst]:
```suggestion
You can also require the ``symfony/deprecation-contracts`` package that provides
a global ``trigger_deprecation()`` function for this usage.
```

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Antoine.

**Pull request merged**